junit-jupiter-api/src/main/java/org/junit/jupiter/api/DisplayNameGenerator.java

```diff
@@ -82,14 +86,41 @@ public interface DisplayNameGenerator {
 	String generateDisplayNameForClass(Class<?> testClass);
 
 	/**
-	 * Generate a display name for the given {@link Nested @Nested} inner test class.
+	 * Generate a display name for the given {@link Nested @Nested} inner test
+	 * class.
 	 *
-	 * <p>If it returns {@code null}, the default display name generator will be used instead.
+	 * <p>If it returns {@code null}, the default display name generator will be
```

Comment:
```suggestion
	 * <p>If this method returns {@code null}, the default display name generator will be
```

Reply:
Done in 70dba48b29de9217bc60de8d65d092dfd9d13fb4